easybuild/easyconfigs/r/READemption/READemption-1.0.5-foss-2021a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonBundle'
+
+name = 'READemption'
+version = '1.0.5'
+
+homepage = 'https://reademption.readthedocs.io/en/latest/index.html'
+description = """READemption is a pipeline for the computational evaluation of RNA-Seq data."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('segemehl', '0.3.4'),
+    ('matplotlib', '3.4.2'),
+    ('Pysam', '0.16.0.1'),
+]
```

Comment:
you're also missing a dependency on `R`, see https://reademption.readthedocs.io/en/latest/installation.html#install-deseq2